zenoh/src/lib.rs

```diff
@@ -351,7 +351,11 @@ pub mod time {
 
 /// Configuration to pass to [`open`] and [`scout`] functions and associated constants
 pub mod config {
-    pub use zenoh_config::{WhatAmI, WhatAmIMatcher};
+    pub use zenoh_config::{EndPoint, Locator, WhatAmI, WhatAmIMatcher, ZenohId};
+    #[zenoh_macros::unstable_config]
+    pub use zenoh_config::{
+        ModeDependentValue, PermissionsConf, PluginsConfig, SecretValue, ValidatedMap,
```

Comment:
How was this chosen? It seems like an odd selection from what's available.

Reply:
I've simply reverted the changes.